Python/Background_Subtractor/BG_Subtractor.py

```diff
@@ -0,0 +1,118 @@
+import cv2
+import numpy as np
+
+# drawn act as a flag variable to see if ROI has been selected or not
+drawn = False
+ix, iy = -1, -1
+# To store ROI values
+rectangle = (0, 0, 0, 0)
+
+
+def open_image(path):
+    """
+    It opens the image when a valid path is given.
+    :param path: image location path
+    :return:
+    """
+    img = cv2.imread(path)
+    dim = (512, 512)
+    resized_image = cv2.resize(img, dim, interpolation=cv2.INTER_AREA)
+    return resized_image
+
+
+# mouse callback function

```

Comment:
Let's remove these comments shall we, you have written an awesome docstring here.